src/config/dynamic-theme-fixes.config

```diff
@@ -8947,6 +8947,15 @@ body,
 
 ================================
 
+mikufan.com
+
+CSS
+#header-container {
+    background-image: url(https://www.mikufan.com/wp-content/uploads/2018/04/header_main3.jpg) !important;
+}
```

Comment:
```suggestion
IGNORE IMAGE ANALYSIS
#header-container
```

Reply:
It didn't work when i tried

Reply:
You've to reload the page before it takes effect.

Reply:
I see, thanks